### PR TITLE
Added 4 new assets

### PR DESCRIPTION
**Pull request by ngwk1**
### Already exist in ignition
- [x] Tree
- [x] Bin 

### Added assets:
- [x] Pond
- [x] Rover Robotics 2WD Rover
- [x] Hedge
- [x] Kiosk 

### To add:
- [ ] Shopping racks/shelves

### CODE REVIEW

**Comment by aaronchongth** [comment]
image
is the robot still a WIP? there are still some missing visual links, the collision looks great though!

File: rmf_demo_assets/models/Hedge/model.sdf [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<?xml version='1.0' encoding='utf-8'?>
+<sdf version="1.6">
+  <model name="Hedge">
```
Comment (aaronchongth) [inline rmf_demo_assets/models/Hedge/model.sdf]: image
The hedges seem to be bit unnaturally green

File: rmf_demo_assets/models/2WDRover/model.sdf [hunk 2]
```diff
@@ -0,0 +1,542 @@
+<sdf version='1.6'>
+  <model name='2WD_rover_robotics'>
```
[the remark this answers is not in the document]
Comment (ngwk1) [reply rmf_demo_assets/models/2WDRover/model.sdf]: this was actually directly imported from the rover robotics, I converted their files and compiled them into a Gazebo asset. I checked the robot they created, seems to have the same visual as well.

**Comment by aaronchongth** [comment]
Closing due to our discussed licensing issues.

**Pull request closed without merging**